### ISSUE TITLE
Can't use custom User

### ISSUE THREAD

**Issue by noodlebreak** [fpytloun/django-xmpp#6]
TO fix:
I changed in models.

```
User = settings.AUTH_USER_MODEL
```

After the imports.

**Comment by fpytloun** [comment]
Thank you for report. Can you please make a pull request with fix?

**Comment by noodlebreak** [comment]
Okay.

**Comment by noodlebreak** [comment]
I just cloned the code and the issue's not there - my fix is already there in place.
I'm not sure why in the version I had installed I had old code.

Edit: The fix is already there, but you haven't pushed the fixed version (0.3.2 - which it is currently) to PyPi.

Please push the new package with fixes to PyPi. And then we can mark this closed.
Currently I'm making do with installing directly from source:

```
pip install git+https://github.com/fpytloun/django-xmpp
```

Thank you.

**Comment by fpytloun** [comment]
Ok, closing then.
